packages/frontend-2/components/workspace/Dashboard.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <WorkspaceProjectList :workspace-id="workspaceId" />
+</template>
+
+<script setup>
+import { useRoute } from 'vue-router'
+
+const route = useRoute()
+const workspaceId = route.params.id
```

Comment:
not reactive
https://www.notion.so/speckle/General-Web-Code-Review-Guide-1f2bf3dfea634460b3778ac3cf4aa1bb?pvs=4#8a1d83812bf24d7ca4fabfa6d521cfd9